cookbooks/ftp/recipes/default.rb

```diff
@@ -51,3 +51,14 @@
   dest_ports "ftp"
   source_ports "-"
 end
+
+# Enable Netfilter ftp helper on port 21
+firewall_rule "helper-ftp-tcp" do
```

Comment:
I don't think this needs (or wants) an extra `HELPER` rule - you can just specify the helper on the existing `ACCEPT` rule by my reading of the manual page.

The ability to use a `HELPER` rule is just there for cases where the packets are already being accepted by the default policy.